### DISCUSSION TITLE
Make a blog about correlation attacks

### DISCUSSION THREAD

**Discussion by jermanuts** [PrivSec-dev/privsec.dev#45] [General]
https://torrentfreak.com/digital-trails-how-bungie-identified-a-mass-sender-of-fake-dmca-notices-220624/
https://www-nrk-no.translate.goog/norge/nye-opplysninger-om-trusselbrevet-i-forsvinningssaken_-_-vet-sa-a-si-alt-om-dette-dokumentet-1.15965322?_x_tr_sl=no&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp
https://serhack.me/articles/unveiling-anonymous-author-stylometry-techniques
https://seirdy.one/notes/2022/06/24/stylometric-fingerprinting-resistance/
https://petsymposium.org/popets/2017/popets-2017-0023.php
https://www.gwern.net/Death-Note-script

IP logs (from ISP) and fingerprinting users, metadata, etc.

**Comment by TommyTran732** [comment]
Maybe. I don't have much to say about it, but if you make a pull request I can review it.

**Reply by jermanuts** [reply]
I updated the post with 2 more links and I see this as a comprehensive list, enough for ctrl c + v a blog post

**Reply by TommyTran732** [reply]
I still do not want to write about things I don't know enough about... you can though and I will review it

**Reply by Seirdy** [reply]
You always have the option of adding a "further research/cool links" section, with a note saying that the topics are WIP or not ready yet and you're still learning about them. "Learning in the open" is a practice that I am very fond of. If you're transparent about what you know *and don't know,* people will trust you much more than if you just publish what you're certain about.

In other words, if you're worried about credibility: sharing topics you're still learning about will only help.

**Reply by jermanuts** [reply]
Agree with seirdy, the amount of effort you put in your site is really appreciated.
Most notably for me are bookmarks/ and notes/

**Reply by jermanuts** [reply]
another 2 links added, all I can find are research papers which I doubt would be fun to add.

**Reply by TommyTran732** [reply]
Well like I said though, I am not entirely ruling this out, but I am not comfortable writing it. Plus, I have a lot of other things that I do know to talk about anyways.

If you can just make the post in a pull request, it would actually help a lot.

**Reply by nopeitsnothing** [reply]
Thanks Seirdy. This topic is very old, but I want to add to this. I also would like to take this PR if nobody else wants to work on it. I'm confident in my knowledge of this particular attack. I have written about it.

**Comment by wj25czxj47bu6q** [comment]
> Thanks Seirdy. This topic is very old, but I want to add to this. I also would like to take this PR if nobody else wants to work on it. I'm confident in my knowledge of this particular attack. I have written about it.

@nopeitsnothing Feel free&nbsp;&mdash; there has been no progress on this particular topic outside of this thread. Just be mindful that we are quite busy with other matters right now, so there may be some delays in reviewing your pull request :slightly_smiling_face:
